<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleWidget</class>
 <widget class="QWidget" name="RoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="role_grid">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="3">
    <widget class="QLabel" name="passwd_lbl">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="passwd_edt"/>
   </item>
   <item row="1" column="2">
    <widget class="QDateTimeEdit" name="validity_dte">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="displayFormat">
      <string>yyyy-MMM-dd hh:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="conn_limit_lbl">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Connections:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="conn_limit_sb">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QGroupBox" name="attributes_gb">
     <property name="title">
      <string>Attributes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="superusr_chk">
        <property name="text">
         <string>Superuser</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="inh_perm_chk">
        <property name="text">
         <string>Inherit Permissions</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="create_db_chk">
        <property name="text">
         <string>Create Database</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="can_login_chk">
        <property name="text">
         <string>Can Login</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="create_user_chk">
        <property name="text">
         <string>Create Users/Groups</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="encrypt_pass_chk">
        <property name="text">
         <string>Encrypted Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QTabWidget" name="members_twg">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="members_tab">
      <attribute name="title">
       <string>Members</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="member_of_tab">
      <attribute name="title">
       <string>Member of</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="adm_members_tab">
      <attribute name="title">
       <string>Members (Admin.)</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="sysid_sb">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="validity_chk">
     <property name="text">
      <string>Validity</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="sysid_lbl">
     <property name="text">
      <string>SysID:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
